wallet/src/ui/app/pages/home/nft-details/index.tsx

```diff
@@ -70,6 +75,7 @@ function NFTdetailsContent({
         </div>
     );
 
+    trackEvent('nft-details');
```

Comment:
Should be in an effect. Also why isn't this just a page view?

Reply:
@Jordan-Mysten can you elaborate where should I put to make it in effect?